python/cudf/pyproject.toml

```diff
@@ -0,0 +1,31 @@
+# Copyright (c) 2021, NVIDIA CORPORATION.
+
+[build-system]
+
+requires = [
+    "wheel",
+    "setuptools",
+    "cython>=0.29,<0.30",
+    "packaging",
+    "cachetools",
```

Comment:
I'm surprised we need `cachetools` or `packaging` here as well. I figured those would be runtime only dependencies.

Reply:
We actually don't need these, removed them